src/default-settings.js

```diff
@@ -48,16 +51,16 @@ const settings = {
             tooltip:null //set in syncSettings()
         }
     ],
-    resizable:true,
-    max_width: 600,
-    margin:{right:20},
-    aspect: 1.33
+    margin:{right:20} //create space for box plot
 };
 
 // Replicate settings in multiple places in the settings object
 export function syncSettings(settings){
     settings.y.column = settings.value_col;
     settings.x.column = settings.time_cols[0];
+    settings.x.type = settings.x.column === 'DY'
```

Comment:
I'm a little worried about having a hard coded variable name here ("DY"). What happens when the user specifies a different set of `time_cols`?  

What if we had time_cols specified like this: `time_cols: [{value_col:"VISITN", axis_type:"ordinal"}, {value_col:"DY", axis_type:"linear}, ...]`, then this area becomes:
```
  settings.x.column = settings.time_cols[0].value_col;
  settings.x.type = settings.x.column =  settings.time_cols[0].axis_type;
```